resource/managing-pipelines-performance.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="managing-pipelines-performance"]
+= Managing {pipelines-shortname} performance
+:context: managing-pipelines-performance
+
+toc::[]
+
+If your {pipelines-shortname} installation runs a large number of tasks at the same time, its performance might degrade. You might experience slowdowns and failed pipeline runs.
+
+For reference, in Red Hat tests, on a three-node {OCP} cluster running on Amazon Web Services (AWS) `m6a.2xlarge` nodes, up to 60 simple test pipelines ran concurrently without significant failures or delays. If more pipelines ran concurrently, the number of failed pipeline runs, the average duration of a pipeline run, the pod creation latency, the work queue depth, and the number of pending pods increased. This testing was performed on {pipelines-title} version 1.13; no statistically significant difference was observed from version 1.12.
```

Comment:
This reference example is confusing, redundant, and maybe unhelpful to anyone not running on AWS with `m6a.2xlarge` nodes. Can this be simplified or eliminated entirely?  You could simply reword line 9 above: 

```
If your {pipelines-shortname} installation runs a large number of tasks at the same time, the {pipelines-shortname}  performance might degrade. Examples of degraded performance might include: 
* failed pipeline runs
* increased average duration of pipeline runs
* increased pod creation latency
* increased work queue depth
* increased pending pods